dist/index.html

```diff
@@ -1,17 +1,42 @@
 <!DOCTYPE html>
 
 <html lang="en">
+  
   <head>
-    <meta charset="utf-8" />
-
-    <title>SHOT RUN</title>
-    <meta name="description" content="SHOT RUN" />
+    <meta charset="UTF-8" />
+    <!-- Update title to "Sober Run" not "Shot Run"-->
+    <title>Sober Run</title>
+    <meta name="description" content="Web Browser Game" />
     <meta name="author" content="Daniel C. Suh and Jaehoon Han" />
+    <!--add a viewport meta tag-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      canvas {
+        display: block;
+      }
+
+      h1 {
+        text-align: center;
+        z-index: 1;
+      }
+    </style>
   </head>
 
   <body>
-    <script src="bundle.js"></script>
-    <h1>Shot... Run!</h1>
-    <canvas width="1000" height="600"></canvas>
+
+    <canvas id = "game">Sober Run</canvas>
+    <!-- commenting out bundle source for now 
+      because it's creating "file does not exist" error
+
+      <script src="dist/bundle.js"></script> -->
+      
+    <script src="../src/index.js"></script>
```

Comment:
i would change this back.
if you run "npm run start", the original code should work